catalogue/webapp/components/Work/Work.tsx

```diff
@@ -41,15 +41,15 @@ const ArchiveDetailsContainer = styled.div`
 
 declare global {
   interface Window {
-    dataLayer: any[];
+    dataLayer: Record<string, unknown>[];
   }
 }
 
 type Props = {
   work: WorkType;
 };
 
-const Work = ({ work }: Props) => {
+const Work = ({ work }: Props): JSX.Element => {
```

Comment:
Should these be `FunctionComponent`s?

Reply:
> A function component is of type `React.FunctionComponent` and it has an alias `React.FC` to keep things nice and short. It has one required property, a function, which will return a `ReactElement` or `null`. It has a few optional properties, such as `propTypes`, `contextTypes`, `defaultProps` and `displayName`.

From [SO](https://stackoverflow.com/a/57363438/590593).

So I think `const Work: FunctionComponent<Props> = ({ work }: Props): ReactElement<Props>` is probably the strictest correct typing.

Reply:
The return type is inferred so you don't have to put it there.

Reply:
Oh, only saw this after I'd added all the `ReactElements`. You're right, of course. Another pr to get rid of them, probably.